src/contracts/erc2477.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.6;
+/**
+ * @title ERC-2477 Non-Fungible Token Metadata Integrity Standard
```

Comment:
@fulldecent what do you think? Is it ok if we include `Non-Fungible` here? Should this be in parenthesis?